rasa/shared/utils/cli.py

```diff
@@ -18,23 +24,48 @@ def print_color(*args: Any, color: Text) -> None:
 
 
 def print_success(*args: Any) -> None:
+    """Print the given arguments to STDOUT in green, indicating success.
+
+    Args:
+        args (:obj:`list` of :obj:`Any`): A list of objects to be printed.
+    """
     print_color(*args, color=rasa.shared.utils.io.bcolors.OKGREEN)
 
 
 def print_info(*args: Any) -> None:
+    """Print the given arguments to STDOUT in blue, indicating an informational
+    message.
```

Comment:
To fix the code quality check
```suggestion
    """Print the given arguments to STDOUT in blue.
    
    Indicate an informational message.
```